components/prompt_based_laion_retrieval/src/main.py

```diff
@@ -34,7 +34,7 @@ def query_clip_client(text: str, client: ClipClient) -> List[str]:
     return urls
 
 
-class LAIONRetrievalComponent(FondantTransformComponent):
+class LAIONRetrievalComponent(WriteComponent):
```

Comment:
a load component is technically also a write component :p

Reply:
Oops, good catch :)